### DISCUSSION TITLE
Quantized models?

### DISCUSSION THREAD

**Discussion by 0wwafa** [Purfview/whisper-standalone-win#356] [Q&A] [answered]
Is there a way to use quantized or distilled models like:

https://huggingface.co/ALM/whisper-it-small

or

https://huggingface.co/Sandiago21/whisper-large-v2-italian  (perhaps quantized to q8_0)

?

**Comment by Purfview** [comment, marked as answer]
Those are not "quantized" nor "distilled ".

If you convert those fine-tuned models to [ctranslate2 ](https://github.com/OpenNMT/CTranslate2)format then you can use them.
